src/onnx/parse_convolution.cpp

```diff
@@ -149,9 +149,40 @@ struct parse_convolution : op_parser<parse_convolution>
 
         recalc_conv_attributes(values, kdims);
 
+        instruction_ref ret;
+        // parse a_zero_point and b_zero_point values
+        auto l0_zp = l0;
```

Comment:
Thanks for doing this PR. 
You don't have to do anything about the pre-existing names ..
This is somehow the simplest and yet trickiest naming and assignment... I can remember..:  Is this 10 or l0, I need a bigger monitor to find that out. :-)